src/test/regress/input/multi_alter_table_statements.source

```diff
@@ -172,16 +172,6 @@ COMMIT;
 SELECT indexname, tablename FROM pg_indexes WHERE tablename = 'lineitem_alter';
 DROP INDEX temp_index_2;
 
--- verify that distributed ddl commands use 2PC, even the value of citus.multi_shard_commit_protocol is '1pc'
```

Comment:
We're losing code coverage for DDL-over-2PC. Could we just SET citus.multi_shard_commit_protocol TO '2pc' ?